### ISSUE TITLE
md-sidenav "below" the app toolbar

### ISSUE THREAD

**Issue by theShadow89** [angular/material2#7881]
#### Bug, feature request, or proposal:

Add mode "below" to sidebar component to show sidebar below the toolbar

#### What is the expected behavior?

show sidebar below the toolbar like [https://material-components-web.appspot.com/drawer/permanent-drawer-below-toolbar.html](https://material-components-web.appspot.com/drawer/permanent-drawer-below-toolbar.html)

#### What is the current behavior?

None

#### What is the use-case or motivation for changing an existing behavior?

New Mode for sidenav that respect the material specification

**Comment by willshowell** [comment]
You don't need a special mode to do it, just adjust your component template so that the toolbar is before your sidenav container.

https://material.angular.io/components/categories does it

**Comment by renrawnalon** [comment]
Hi @willshowell,

I'm having a similar issue in Material 2 that I haven't been able to resolve, where the toolbar is above the side menu, but it scrolls with the content if it is taller than the page content rather than remaining fixed at the top of the window. In Material 1, using `md-content` seemed to solve this allowing content to scroll independently of the toolbar. However, I haven't been able to find an equivalent to `md-content` in Material 2 mimic this desired behavior. I looked at the sidenav sample code in the demo-app, and I saw the same thing happening. Am I missing something?
